proctor-common/src/main/java/com/indeed/proctor/common/model/ConsumableTestDefinition.java

```diff
@@ -131,7 +133,8 @@ private ConsumableTestDefinition(
             @Nonnull final Map<String, Object> constants,
             @Nullable final String description,
             @Nonnull final List<String> metaTags,
-            @Nullable final TestDependency dependsOn) {
+            @Nullable final TestDependency dependsOn,
+            final boolean enableUnitlessAllocations) {
```

Comment:
Just to confirm, is it going to be backward compatible with old test definitions which don't have this field ( and should default to false) ?

Reply:
Yes, if test do not have the field it will be loaded as false. Added some unit tests to confirm